connector/connect/client/jvm/src/main/scala/org/apache/spark/sql/Dataset.scala

```diff
@@ -1906,6 +1906,100 @@ class Dataset[T] private[sql] (val session: SparkSession, private[sql] val plan:
     }
   }
 
+  /**
+   * Registers this Dataset as a temporary table using the given name. The lifetime of this
+   * temporary table is tied to the [[SparkSession]] that was used to create this Dataset.
+   *
+   * @group basic
+   * @since 3.4.0
+   */
+  @deprecated("Use createOrReplaceTempView(viewName) instead.", "2.0.0")
```

Comment:
BTW .. we should probably grep and replace these versions in deprecation (2.0.0 in this case) to 3.4.0  .. of course I don't mind doing separately.

Reply:
Updated.

I didn't update it was because this API is added from `3.4.0` for Connect but deprecated since `3.4.0`.